java: Generate API proto only test fails

```
--- FAIL: TestGenerateAPI_ProtoOnly (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered, repanicked]
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xb66bbe]

goroutine 62 [running]:
testing.tRunner.func1.2({0xcb9120, 0x12ab620})
	/opt/hostedtoolcache/go/1.26.1/x64/src/testing/testing.go:1974 +0x419
testing.tRunner.func1()
	/opt/hostedtoolcache/go/1.26.1/x64/src/testing/testing.go:1977 +0x67f
panic({0xcb9120?, 0x12ab620?})
	/opt/hostedtoolcache/go/1.26.1/x64/src/runtime/panic.go:860 +0x13a
github.com/googleapis/librarian/internal/librarian/java.generateAPI({0xda0af8, 0xc00055a410}, 0xc000118380, 0xc000133d90, 0xc0003b4500, {0xd6fd56, 0x19}, {0xc0005d2360, 0x2c}, 0xc0003da6e0, ...)
	/home/runner/work/librarian/librarian/internal/librarian/java/generate.go:102 +0x57e
github.com/googleapis/librarian/internal/librarian/java.TestGenerateAPI_ProtoOnly(0xc00055f8c8)
	/home/runner/work/librarian/librarian/internal/librarian/java/generate_test.go:398 +0xc9b
testing.tRunner(0xc00055f8c8, 0xd93c00)
	/opt/hostedtoolcache/go/1.26.1/x64/src/testing/testing.go:2036 +0x21d
created by testing.(*T).Run in goroutine 1
	/opt/hostedtoolcache/go/1.26.1/x64/src/testing/testing.go:2101 +0xb13
FAIL	github.com/googleapis/librarian/internal/librarian/java	0.912s
FAIL
2026/04/14 00:29:12 go test: /opt/hostedtoolcache/go/1.26.1/x64/bin/go test -race -coverprofile=coverage.out -covermode=atomic ./internal/librarian/java: exit status 1
exit status 1
Error: Process completed with exit code 1.
```

https://github.com/googleapis/librarian/actions/runs/24374152019/job/71184220269

--- a/internal/librarian/java/generate_test.go
+++ b/internal/librarian/java/generate_test.go
@@ -380,6 +380,9 @@ func TestGenerateAPI_ProtoOnly(t *testing.T) {
 	library := &config.Library{
 		Name:   "gkehub",
 		Output: outdir,
+		APIs: []*config.API{
+			{Path: "google/cloud/gkehub/policycontroller/v1beta"},
+		},
 		Java: &config.JavaModule{
 			JavaAPIs: []*config.JavaAPI{
 				{Path: "google/cloud/gkehub/policycontroller/v1beta", ProtoOnly: true},

--- a/internal/librarian/java/generate_test.go
+++ b/internal/librarian/java/generate_test.go
@@ -386,6 +386,9 @@ func TestGenerateAPI_ProtoOnly(t *testing.T) {
 			},
 		},
 	}
+	if _, err := Fill(library); err != nil {
+		t.Fatal(err)
+	}
 	for _, artifact := range []string{"proto-google-cloud-gkehub-v1beta", "google-cloud-gkehub-bom"} {
 		if err := os.MkdirAll(filepath.Join(outdir, artifact), 0755); err != nil {
 			t.Fatal(err)

